modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/notifications/ScheduledPublicationUserNotificationHandler.java

```diff
@@ -100,9 +126,39 @@ protected String getLink(
 			).buildString();
 		}
 
+		if (Objects.equals(
+				userNotificationEvent.getUserId(),
+				_getAdminUserId(userNotificationEvent.getCompanyId()))) {
+
+			return PortletURLBuilder.create(
+				_portal.getControlPanelPortletURL(
+					serviceContext.getRequest(), serviceContext.getScopeGroup(),
+					CTPortletKeys.PUBLICATIONS, 0, 0,
+					PortletRequest.RENDER_PHASE)
+			).setMVCRenderCommandName(
+				"/change_tracking/view_stack_trace"
+			).setParameter(
+				"backgroundTaskId",
+				_jsonFactory.createJSONObject(
+					userNotificationEvent.getPayload()
+				).getLong(
+					"backgroundTaskId"
+				)
+			).buildString();
+		}
+
 		return null;
 	}
 
+	private long _getAdminUserId(long companyId) throws Exception {
+		Role role = _roleLocalService.getRole(
+			companyId, RoleConstants.ADMINISTRATOR);
+
+		long[] userIds = _userLocalService.getRoleUserIds(role.getRoleId());
+
+		return userIds[0];
+	}
```

Comment:
Update this method to return a boolean and use `_userLocalService.hasRoleUser(role.getRoleId(), userId)` to check if recipient is an admin.